components/ml_metrics/Calculate_classification_metrics/_samples/sample_pipleine.py

```diff
@@ -0,0 +1,93 @@
+from pathlib import Path
+
+import kfp
+from kfp.components import ComponentStore, create_component_from_func, InputPath, OutputPath
+
+store = ComponentStore.default_store
+store.local_search_paths.append(str(Path(__file__).parent.parent.parent.parent))
```

Comment:
You might want to avoid making modifications to the default store.

Reply:
I've changed the logic to explicitly load the component instead of modifying the store.